extensions/vscode-api-tests/src/singlefolder-tests/terminal.shellIntegration.test.ts

```diff
@@ -87,6 +87,16 @@ import { assertNoRpc } from '../utils';
 		await closeTerminalAsync(terminal);
 	});
 
+	if (platform() === 'darwin') {
```

Comment:
should this be linux too?

Reply:
Yes actually, since the test should work on Linux, it's just the suite is only enabled on mac atm

Reply:
Yes actually, since the test should work on Linux, it's just the suite is only enabled on mac atm

Reply:
Changes right here: https://github.com/microsoft/vscode/pull/237602/commits/5560bdc72cb3d1d95643d31706d660f0abc21859